docs/platform/atoms/booker.mdx

```diff
@@ -77,7 +77,7 @@ For a demonstration of the booker atom along with calendar overlay, please refer
 
 <p></p>
 
-We offer all kinds of customizations to the booker atom via props and customClassNames. 
+We offer all kinds of customizations to the booker atom via props and customClassNames.
```

Comment:
```suggestion
We offer all kinds of customizations to the booker atom via props and customClassNames.
```
can't see any change?

Reply:
Only a space was removed with the linter